docker/imposter/security/security-me-policies.js

```diff
@@ -0,0 +1,310 @@
+var userPoliciesMap = {
+  PLUGIN_SECURITY_USERS_ALL: {
+    'security:read': {
+      'user:id:*': 'allow',
+      'role:id:*': 'allow',
+    },
+    rbac_mode: 'black',
+  },
+  PLUGIN_SECURITY_USERS_DENY_USERS: {
+    'security:read': {
+      'user:id:*': 'deny',
+      'role:id:*': 'allow',
+    },
+    rbac_mode: 'black',
+  },
+  PLUGIN_SECURITY_USERS_DENY_ROLES: {
+    'security:read': {
+      'user:id:*': 'allow',
+      'role:id:*': 'deny',
+    },
+    rbac_mode: 'black',
+  },
+  PLUGIN_SECURITY_USERS_DENY_USERS_ROLES: {
+    'security:read': {
+      'user:id:*': 'deny',
+      'role:id:*': 'deny',
+    },
+    rbac_mode: 'black',
+  },
+  PLUGIN_SECURITY_USERS_DENY_CREATE_USER: {
+    'security:read': {
+      'user:id:*': 'allow',
+      'role:id:*': 'allow',
+    },
+    'security:create_user': {
+      '*:*:*': 'deny',
+    },
+    rbac_mode: 'black',
+  },
+  PLUGIN_SECURITY_USERS_ALLOW_CREATE_USER_DENY_EDIT_RUN_AS: {
+    'security:read': {
+      'user:id:*': 'allow',
+      'role:id:*': 'allow',
+    },
+    'security:create_user': {
```

Comment:
***
Using an user with security:read(userid*):allow, security:read(roleid*):allow and security:update(userid{user_id}):allow and security:edit_run_as(*:*:*):allow should be allowed to edit run as option of user in Security > Users.
***

For this test, I don't know if this is missing or should be changed.

Reply:
Did you fix it the mock API responses locally?

Reply:
Yes, but I see you already added it, good job!